### PR TITLE
added steps

### PR DESCRIPTION
**Pull request by JV0812**
added step 1, 2, and 3 as  per the ticket.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 1]
```diff
@@ -12,9 +12,47 @@ Given that we use an account alias, we recommend you use StackSets to automati
 
 ## Before you start
 
-You need to install apps and determine account aliases before deploying.
+If this is the first time you've deployed the AWS Observability Solution, read the [Before You Deploy](https://help.sumologic.com/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:15]:
```suggestion
If this is the first time you've deployed the AWS Observability solution, read the [Before You Deploy](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information.
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 2]
```diff
@@ -12,9 +12,47 @@ Given that we use an account alias, we recommend you use StackSets to automati
...
 
-## Step 1: Install Apps
+## Step 1: Open the CloudFormation template
+
+1. Sign on to the AWS Management console.
+1. Choose an option:
+   * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml)
+   * Click the AWS Observability Solution (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template. If you would like to download or inspect this or other versions of this template, please visit [Change Log](../changelog.md).
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:22]: We don't use 'please'

```suggestion
   * Click the AWS Observability solution (S3 Link for CloudFormation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template. If you would like to download or inspect this or other versions of this template, see [Change Log](../changelog.md).
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 3]
```diff
@@ -48,7 +86,7 @@ In case you do not provide a CSV file or if we detect that it does not have the
     in your AWS account.
 1. Click **Create StackSet**.  
     ![CloudFormation_Stackset 1.png](/img/observability/ClodFormation_Stackset_1.png)
-1. Paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.5.1/sumologic_observability.master.template.yaml` in the Amazon S3 URL option and click **Next**. If you would like to download or inspect this or other versions of this template, please visit the [change log](../changelog.md) page.<br/>  ![multi-create-stack.png](/img/observability/multi-create-stack.png)
+1. Paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option and click **Next**. If you would like to download or inspect this or other versions of this template, please visit the [change log](../changelog.md) page.<br/>  ![multi-create-stack.png](/img/observability/multi-create-stack.png)
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:89]:
```suggestion
1. Paste the URL `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml` in the Amazon S3 URL option and click **Next**. If you'd like to download or inspect this or other versions of this template, see the [Changelog](../changelog.md) page.<br/>  ![multi-create-stack.png](/img/observability/multi-create-stack.png)
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Grammar, formatting

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 4]
```diff
@@ -26,7 +63,7 @@ You need to install apps and determine account aliases before deploying.
 1. Enable ELB Classic Access logging as **None** and Create Sumo Logic ELB Classic Logs Source as **No**. <br/>  ![Multiaccount6.png](/img/observability/Multiaccount6.png)
 1. Location where you want the App to be Installed as **PersonalFolder**. And for **Do you want to share App with whole organization**, set as **True**.<br/>  ![Multiaccount7.png](/img/observability/Multiaccount7.png)
 
-## Step 2: Determine Account Aliases
+## Step 5: Determine Account Aliases
```
Comment (JainM6) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:66]:
```suggestion
## Step 5: Determine account aliases
```

**Review by JainM6 — APPROVED** [review]
Looks good to me.

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ If this is the first time you've deployed the AWS Observability solution, read t
 
 1. Sign on to the AWS Management console.
 1. Choose an option:
```
Comment (JainM6) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:20]: Can we write why the user needs to choose the option?

Comment (JV0812) [reply docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md]: updated

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 6]
```diff
@@ -12,9 +12,47 @@ Given that we use an account alias, we recommend you use StackSets to automati
 
 ## Before you start
 
-You need to install apps and determine account aliases before deploying.
+If this is the first time you've deployed the AWS Observability solution, read the [Before You Deploy](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information. And, complete the prerequisites for StackSets as described in the [AWS documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:15]:
```suggestion
* If this is the first time you've deployed our AWS Observability solution, read the [Before You Deploy](/docs/observability/aws/deploy-use-aws-observability/before-you-deploy/) topic for more information.
* Complete the prerequisites for StackSets as described in the [AWS documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs.html).
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 7]
```diff
@@ -12,9 +12,47 @@ Given that we use an account alias, we recommend you use StackSets to automati
...
 
-## Step 1: Install Apps
+## Step 1: Open the CloudFormation template
+
+1. Sign on to the AWS Management console.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:19]:
```suggestion
1. Sign in to the AWS Management console.
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md [hunk 8]
```diff
@@ -12,9 +12,47 @@ Given that we use an account alias, we recommend you use StackSets to automati
...
+1. Choose an option to invoke AWS CloudFormation Template:
+    * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml) to invoke the latest Sumo Logic AWS CloudFormation template.
+    * Download the AWS Observability Solution template (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.
+    :::note
+    If you would like to download or inspect this or other versions of this template, see the [Changelog](../changelog.md).
+    :::
+1. Select the AWS Region where you want to deploy the AWS CloudFormation template.
+    :::warning
+    This step is critical: if you do not select the correct region, you will deploy the solution in the wrong region.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/deploy-multiple-accounts-regions.md:28]:
```suggestion
    This step is critical. If you do not select the correct region, you will deploy the solution in the wrong region.
```

**Review by kimsauce — DISMISSED** [review]
minor fixes

**Review by JainM6 — APPROVED** [review]
LTGM

**Review by JainM6 — APPROVED** [review]
LTGM

**Pull request merged**